docs/DMS/DATA-STORAGE-SCHEMA-GEN.md

```diff
@@ -0,0 +1,63 @@
+# DMS Feature: Data Storage Schema Generation and Deployment
+
+## Problems to solve
+
+### Ease of DB bootstrapping
+
+We would like for DB schema creation to be either a zero-step or at most one-step process. Meadowlark was
+designed with a zero-step process, meaning that if the database did not exist it was created automatically.
+This was of major benefit both for testing and onboarding new users.
+
+### Responsibility for and time of schema generation
+
+We need to decide which parts of the Tanager system will have the responsibility of creating the database
+schema, and at what stage of the deployment lifecycle generation will occur. The major system division is
+between MetaEd, which performs pre-processing for Tanager, and Tanager itself. Example lifecycle stages
+include software development time, pre-processing time, and runtime.
```

Comment:
Suggest revising to remove "Tanager" from the paragraph. Once Project Tanager is "done" (production-ready), this documentation probably moves elsewhere and "Tanager" likely isn't part of that context.

Reply:
Done